aiida/cmdline/utils/echo.py

```diff
@@ -196,6 +181,25 @@ def default_jsondump(data):
     return json.dumps(dictionary, indent=4, sort_keys=True, default=default_jsondump)
 
 
+VALID_DICT_FORMATS_MAPPING = OrderedDict((('json+date', _format_dictionary_json_date), ('yaml', serialize)))
+
+
+def echo_dictionary(dictionary, fmt='json+date'):
+    """
+    Print the given dictionary to stdout in the given format
+
+    :param dictionary: the dictionary
+    :param fmt: the format to use for printing, valid options: ['json+data']
```

Comment:
typo in `'json+data'` but also incorrect, right? `yaml` is also a valid option? Did you mean that is the default perhaps?